<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>771</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>首页</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>全部商品</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>关于我们</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="user_info_comboBox">
     <property name="geometry">
      <rect>
       <x>379</x>
       <y>10</y>
       <width>171</width>
       <height>36</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="user_img_label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="favorite_label">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>63</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>我的收藏</string>
     </property>
    </widget>
    <widget class="QLabel" name="order_label">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>63</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>我的订单</string>
     </property>
    </widget>
    <widget class="QLabel" name="cart_label">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>63</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>购物车</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="search_edit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>171</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
